src/components/Profile/Profile.jsx

```diff
@@ -1,219 +1,148 @@
-              ) : (
-                <SvgAvatar>
-                  <use href={Icon + '#icon-ph-user'}></use>
-                </SvgAvatar>
-              )}
-
-              <LabelImg htmlFor="avatar">
-
-                <InputFile
-                  id="avatar"
-                  type="file"
-                  onChange={event => setAvatarURL(event.target.files[0])}
-                  accept="image/png, image/gif, image/jpeg"
-                  name="avatar"
-                ></InputFile>
-
-                <ErrorMessage name="avatar">
-                  {msg => {
-                    <Message>{msg}</Message>;
-                  }}
-                </ErrorMessage>
-              </LabelImg>
-            </ContainerImg> */}
-
-            <h2>{user?.name} </h2>
-            <User>User</User>
-            <BlockInput>
-              <LabelBtn htmlFor="name">
-                <p>User Name</p>
-                <Input
-                  type="text"
-                  name="name"
-                  id="name"
-                  value={values.name}
-                  onChange={handleChange}
-                  onBlur={handleBlur}
-                  placeholder="Name"
-                />
-                <ErrorMessage name="name">
-                  {msg => <Message>{msg}</Message>}
-                </ErrorMessage>
-              </LabelBtn>
-
-              {/* <LabelBtn htmlFor="phone">
-                <p>Phone</p>
-                <Input
-                  type="tel"
-                  name="phone"
-                  id="phone"
-                  value={values.phone ? values.phone : ''}
-                  onChange={handleChange}
-                  onBlur={handleBlur}
-                  placeholder="+380"
-                ></Input>
-                <ErrorMessage name="phone">
-                  {msg => <Message>{msg}</Message>}
-                </ErrorMessage>
-              </LabelBtn> */}
-
-              {/* <LabelBtn htmlFor="birthday">
-                <p>Birthday</p>
-                <DatePick
-                  type="date"
-                  name="birthday"
-                  id="birthday"
-                  input={true}
-                  maxDate={new Date()}
-                  selected={values.birthday}
-                  onChange={e => {
-                    setFieldValue('birthday', e);
-                    setTouched({ ...touched, birthday: true });
-                  }}
-                  placeholder="Birthday"
-                  dateFormat="dd/MM/yyyy"
-                /> */}
+  const cancelEdit = () => {
+    setEditMode(false);
+  };
 
-              {/* <VectorPng>
-                  <use href={Icon + '#icon-chevron-right'}></use>
-                </VectorPng>
+  const formik = useFormik({
+    initialValues: user,
+    validationSchema: schema,
+    validateOnChange: false,
+    enableReinitialize: true,
+    onSubmit: async (values, { resetForm }) => {
+      // console.log('SUBMIT', values);
```

Comment:
Remove all console.log before the push your request

Reply:
done